src/main/java/bio/terra/janitor/service/cleanup/FlightScheduler.java

```diff
@@ -113,6 +122,13 @@ private void completeFatalFlights() {
     logger.info("Done completing {} fatal flights.", completedFlights);
   }
 
+  private void recordResourceCount() {
```

Comment:
Optional: Is that hard to verify those metrics in scheduler test or some other test for metrics? If that is hard to setup, that is fine.

Reply:
Added a test. I guess it verifies that we're scheduling the periodic recording, but I'm not totally sold on the value of it. PTAL